lib/loaders/typescript.js

```diff
@@ -32,6 +32,17 @@ module.exports = {
             [config]
         );
 
+        // fork-ts-checker-webpack-plugin integration
+        if (webpackConfig.useForkedTypeScriptTypeChecking) {
+            loaderFeatures.ensureLoaderPackagesExist('forkedtypecheck');
+            // force transpileOnly to speed up
+            config.transpileOnly = true;
+
+            // add forked ts types plugin to the stack
+            const forkedTypesPluginUtil = require('../plugins/forked-ts-types');
+            forkedTypesPluginUtil(webpackConfig);
+        }
```

Comment:
It's in #116 - we'll probably merge this first. If so, I'll add the warning after #116